App.tsx

```diff
@@ -57,23 +58,15 @@ const App = () => {
     // RN version can be displayed.
     SplashScreen.hide()
     initAgent()
-
-    setTimeout(() => {
-      // The app loads quite fast, this delay is to allow for a more
-      // graceful transition.
-      setLoading(false)
-    }, 2000)
-  }, [loading])
-
-  if (loading) {
-    return <Splash />
-  }
+  }, [])
 
   return (
-    <AgentProvider agent={agent}>
-      <RootStack />
-      <Toast topOffset={15} config={toastConfig} />
-    </AgentProvider>
+    <Store>
```

Comment:
Small nitpick: for consistency, can we call this `StoreProvider`?

Reply:
Fixed.